### PR TITLE
Update UG tips and warnings

### PR DESCRIPTION
**Pull request by jordanyoong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -223,6 +223,25 @@ You can use this command when you want to add an entry (expense/revenue) to the
 
 </div>
 
+<div markdown="block" class="alert alert-danger">
+
+:warning: **Warning:**
+
```
Comment (Ziyang-98) [inline docs/UserGuide.md]: I think this is more of a note instead of a warning

File: docs/UserGuide.md [hunk 2]
```diff
@@ -380,6 +414,15 @@ entries, use the `delete` command instead.
...
```
Comment (Ziyang-98) [inline docs/UserGuide.md]: This is solved already. Clear command clears only entries filtered.

**Review by Ziyang-98 — COMMENTED** [review]
Check my comments!

File: docs/UserGuide.md (hunk 1 above)
Comment (jordanyoong) [reply docs/UserGuide.md]: haha actl i put under notes, but the doc was under warning but ok i'll change it back

Comment (Ziyang-98) [reply docs/UserGuide.md]: ohh okok can, warnings are for like irreversible effects, like delete account

**Review by lettuceman4 — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -239,6 +251,21 @@ it anymore.
...
+:bulb: **Tip:**
+
+* It is possible for you to use the `find` function then `delete`.
+
+</div>
+
+:warning: **Warning:**
+
+* Entry index should only be a single integer, for example, `delete 1 2 c/revenue` will not work.
```
Comment (Ziyang-98) [inline docs/UserGuide.md:264]: This should be a note uh

Comment (Ziyang-98) [reply docs/UserGuide.md:264]: delete shouldn't have an irreversible effect since there is an undo command

**Pull request merged**